tests/e2e/enclave-cc-pod-sim.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: enclave-cc-pod-sim
+spec:
+  runtimeClassName: enclave-cc
+  containers:
+  - name: hello-world
+    image: docker.io/huaijin20191223/scratch-base:v1.8
+    imagePullPolicy: IfNotPresent
+    env:
+    - name: OCCLUM_RELEASE_ENCLAVE
+      value: "0"
+    workingDir: "/run/rune/boot_instance/"
+    command:
+    - /run/rune/boot_instance/build/bin/occlum-run
+    - /bin/hello_world
```

Comment:
@mythi question for my knowledge. Does `sim` refers to simulator ?

Reply:
@bpradipt, yes, SIM refers to SIMulator. :-)

The other flavour of Enclave CC is HW (hardware), unfortunataly we cannot have both modes running at the same time.
Having this Simulator mode enabled allows folks testing Enclave CC even without access to a SGX capable machine.

Reply:
Nice, I was wondering how to tryout enclave cc and this looks cool. Will try this out